modules/mailer/mailer.go

```diff
@@ -72,9 +72,15 @@ func sendMail(settings *setting.Mailer, recipients []string, msgContent []byte)
 		return err
 	}
 
+	cert, err := tls.LoadX509KeyPair(settings.CertFile, settings.KeyFile)
```

Comment:
Should we add a line before this to check if user is willing to use cert and key?

The code right now looks like it forces user to have cert and key files.